regrippy/plugins/usersids.py

```diff
@@ -0,0 +1,40 @@
+# Plugin written by Tim Taylor, timtaylor3@yahoo.com
+from regrippy import BasePlugin, PluginResult, mactime
+
+
+class Plugin(BasePlugin):
+    """Returns the user names with sids"""
+
+    __REGHIVE__ = "SOFTWARE"
+
+    def run(self):
+        path = r"Microsoft\Windows NT\CurrentVersion\ProfileList"
+        key = self.open_key(path)
+        if not key:
+            print('Key not found')
```

Comment:
If the key doesn't exist, `BasePlugin.open_key()` will automatically log the error (though it's only shown if the "verbose" flag is active). This line can be safely removed :)
```suggestion
```